provider/aci_test.go

```diff
@@ -852,6 +852,35 @@ func ptrQuantity(q resource.Quantity) *resource.Quantity {
 	return &q
 }
 
+func TestConfigureNode(t *testing.T) {
+	_, _, provider, err := prepareMocks()
+	if err != nil {
+		t.Fatal("Unable to prepare the mocks", err)
+	}
+
+	node := &v1.Node{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: "virtual-kubelet",
+			Labels: map[string]string{
+				"type":                   "virtual-kubelet",
+				"kubernetes.io/role":     "agent",
+				"kubernetes.io/hostname": "virtual-kubelet",
+			},
+		},
+		Spec: v1.NodeSpec{},
+		Status: v1.NodeStatus{
+			NodeInfo: v1.NodeSystemInfo{
+				Architecture:   "amd64",
+				KubeletVersion: "1.18.4",
```

Comment:
Are we always just using a single version for these mocks?

Reply:
the version here actually doesn't matter for tests.